ipaclient/install/client.py

```diff
@@ -3132,6 +3132,13 @@ def uninstall(options):
     remove_file(paths.SSSD_MC_GROUP)
     remove_file(paths.SSSD_MC_PASSWD)
 
+    try:
+        run([paths.SSSCTL, "cache-remove", "-o", "--stop", "--start"])
+    except Exception:
+        logger.info(
+            "sssd cache should be removed before using"
```

Comment:
As far as I known, ``logger.info()`` strings are not visible to the user and are only logged to the uninstaller log.

Reply:
Thanks for review.
But i feel logger.info() is visible to user on console itself.
As I see in code:
logger.info("Unenrolling client from IPA server")   //seen on console

logger.info(
            "Removing Kerberos service principals from /etc/krb5.keytab")  //seen on console.